kem/frodo/frodo640shake/frodo.go

```diff
@@ -1,4 +1,5 @@
 // Package frodo640shake implements the variant FrodoKEM-640 with SHAKE.
+// Multi-dimensional arrays are stored in 1-dimensional arrays in row-major order.
```

Comment:
Better to move that down to the `type` block where those multi-dimensional types are defined.